cudax/include/cuda/experimental/__device/attributes.cuh

```diff
@@ -32,31 +32,8 @@ namespace cuda::experimental
 namespace detail
 {
 
-_CCCL_NODISCARD inline int __get_attr_impl(::cudaDeviceAttr __attr, int __dev)
-{
-  int __value = 0;
-  _CCCL_TRY_CUDA_API(::cudaDeviceGetAttribute, "failed to get device attribute", &__value, __attr, __dev);
-  return __value;
-}
-
-template <::cudaDeviceAttr _Attr>
-struct __dev_attr
```

Comment:
Critical: this breaks with the forward declaration in device_ref:

https://github.com/NVIDIA/cccl/blob/7ff1d7bde11065ecd8328395e4e5e84191bc18d7/cudax/include/cuda/experimental/__device/device_ref.cuh#L36-L40

That specializes this template.

Reply:
We still define the same template:

```cpp
template <::cudaDeviceAttr _Attr>
struct __dev_attr : __dev_attr_impl<_Attr, int>
{};
```

Reply:
First PRs on Monday .. Thx!